src/components/DialogmoteInfoPanel/DialogmoteSykmeldingerInfoPanel.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import { Link } from '@navikt/ds-react'
+
+import { RootState } from '../../state/store'
+import Veileder from '../shared/veileder/Veileder'
+import { getPublicEnv } from '../../utils/env'
+import { logAmplitudeEvent } from '../../amplitude/amplitude'
+
+const publicEnv = getPublicEnv()
+
+interface Props {
+    sykmeldtId: string
+    name: string
+}
+
+function DialogmoteSykmeldingerInfoPanel({ sykmeldtId, name }: Props): JSX.Element | null {
+    const dialogmoteFeature = useSelector((state: RootState) => state.metadata.dialogmoteFeature)
+    const hasDismissed = localStorage.getItem('dialogmote-sykmeldinger-info') ?? 'false'
+    const shouldShow = dialogmoteFeature !== 'sykmeldinger' || hasDismissed === 'true'
```

Comment:
Panelet skal visast på sida for sykmeldinger?

Reply:
huh.. har jeg snudd på de? :sweat: